README.md

```diff
@@ -95,11 +95,7 @@ The [axe-core API](doc/API.md) package consists of:
 
 Axe can be built using your local language. To do so, a localization file must be added to the `./locales` directory. This file must have be named in the following manner: `<langcode>.json`. To build axe using this locale, instead of the default, run axe with the `--lang` flag, like so:
 
-`grunt build --lang=nl`
-
-or equivalently:
-
-`npm run build -- --lang=nl`
+`grunt build -- --lang=nl`
```

Comment:
We should keep the `npm` command in the docs since `grunt` only works if you have it installed globally. In fact, we should probably update the docs to either use the `npm` command or use `npx grunt` so these grunt commands work (if there's not an npm script available).

Reply:
Oh you know, I thought the line I removed was `npm run build --lang=nl`, not using grunt. The issue it seems users were having is trying to run `npm run build` without having those two extra dashes.